packages/@webex/plugin-logger/test/unit/spec/logger.js

```diff
@@ -185,6 +185,52 @@ describe('plugin-logger', () => {
       assert.lengthOf(webex.logger.buffer, 1);
       assert.match(webex.logger.buffer[0][3], /WebexHttpError/g);
     });
+
+    it('formats objects as strings passed to the logger for readability not [Object object]', async () => {
+      webex.config.logger.level = 'trace';
+      const obj = {
+        headers: {
+          authorization: 'Bearer',
+          trackingid: '123',
+        },
+        test: 'object',
+        nested: {
+          test2: 'object2',
+        }
+      }
+
+      webex.logger.log('foo', 'bar', obj);
+      assert.lengthOf(webex.logger.buffer, 1);
+      assert.lengthOf(webex.logger.buffer[0], 6);
+      assert.deepEqual(webex.logger.buffer[0][2], 'wx-js-sdk');
+      assert.deepEqual(webex.logger.buffer[0][3], 'foo');
+      assert.deepEqual(webex.logger.buffer[0][4], 'bar');
+      assert.deepEqual(webex.logger.buffer[0][5], '{"headers":{"trackingid":"123"},"test":"object","nested":{"test2":"object2"}}');
+    });
+
+    it('formats objects as strings passed to the logger for readability not [Object object] w/ circular reference', async () => {
+      webex.config.logger.level = 'trace';
+      const obj = {
+        headers: {
+          authorization: 'Bearer',
+          trackingid: '123',
+        },
+        test: 'object',
+        nested: {
+          test2: 'object2',
+        }
+      }
+
+      obj.selfReference = obj;
```

Comment:
maybe it would be worth also adding a test case for the circular reference being inside some array? and also for it to reference some other node in the 'obj', not just the `obj` itself